app/backend/src/couchers/standardized_queries.py

```diff
@@ -0,0 +1,7 @@
+from sqlalchemy import not_
+
+from couchers.models import User
+
+
+def query_users_who_arent_hidden(session):
+    return session.query(User).filter(not_(User.is_hidden_for_sql))
```

Comment:
This should be left at the query site, as not all queries use exactly this same pattern (e.g. when joining from non-User table to User).

Also, you can simplify the not by writing `(~User.is_hidden)` instead